nemo_curator/utils/distributed_utils.py

```diff
@@ -33,7 +33,13 @@
 from nemo_curator.utils.import_utils import gpu_only_import, gpu_only_import_from
 
 cudf = gpu_only_import("cudf")
-LocalCUDACluster = gpu_only_import_from("dask_cuda", "LocalCUDACluster")
+
+
+if TYPE_CHECKING:
+    from dask_cuda import LocalCUDACluster
+else:
```

Comment:
Do not think we should duplicate importing to enable type checking and also loose out on  `gpu_only_import_from` functionality

We should long term enable this in the  gpu_only_import_from directly.